docs/internals/source_mappings.rst

```diff
@@ -1,70 +1,71 @@
-.. index:: source mappings
+.. index:: asignaciones de origen
```

Comment:
Please don't translate the index entry here

Reply:
Hey @elijahhampton I've made the change, let me know if there's anything else.